pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -442,7 +442,13 @@ func (p *staticPolicy) guaranteedCPUs(pod *v1.Pod, container *v1.Container) int
 	// We should return this value because this is what kubelet agreed to allocate for the container
 	// and the value configured with runtime.
 	if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
-		if cs, ok := podutil.GetContainerStatus(pod.Status.ContainerStatuses, container.Name); ok {
+		containerStatuses := pod.Status.ContainerStatuses
+		//if utilfeature.DefaultFeatureGate.Enabled(features.SidecarContainers) && types.IsRestartableInitContainer(container) {
+		//if len(pod.Status.InitContainerStatuses) != 0 {
+		containerStatuses = append(containerStatuses, pod.Status.InitContainerStatuses...)
+		//}
+		//}
```

Comment:
Cleanup.

Static CPU & memory policy is out of scope for the beta (https://github.com/kubernetes/kubernetes/issues/128068), so you can just leave a TODO here for now.